Examples/xc.authentication/build-osx.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+##########################################################################
+# This is the Cake bootstrapper script for Linux and OS X.
```

Comment:
>  This is the Cake bootstrapper script for Linux and OS X.

why do we need two scripts?

Reply:
I don't know, I just took it as it is from koordinator

Reply:
Cake provides a script for Linux and a script for OS X. But the comment in the header is the same.